src/components/tests/List.spec.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import List from "../List";
+import mockedData from '../../youtube-videos-mock.json';
+
+describe("items list", () => {
+  it("There is an items list displayed", () => {
+    const { items } = mockedData;     
+    render(<List items={items} />);
+    expect(screen.queryByTestId("items-list").tagName).toBe('DIV');
+    expect(screen.queryAllByTestId("items-list").length).toBe(1);
+  });
+  it("There are 25 items in the items list", () => {
+    const { items } = mockedData;     
+    render(<List items={items} />);
+    expect(screen.queryAllByTestId("item").length).toBe(25);
```

Comment:
with `<List>` you're rendering `<Card>`s and in consequence tags: `<h4>` and `<img>` which are reachable by role: `heading` and `image` respectively. It would be a more effective test if you assert against them and reserve the usage of test-id props to more complex or embedded not-easy-to-reach HTML elements